stories/Introduction.stories.mdx

```diff
@@ -0,0 +1,14 @@
+<Meta title="Introduction" />
```

Comment:
Could we also add a `README.md` file to `stories/` that shows what commands need to be run? Similar to https://github.com/internetarchive/openlibrary/blob/master/openlibrary/components/README.md ? The main questions I think would be nice to answer there would be:

- What's the flow I should use when developing a component? Does storybook have a live-reloading thing I can use, or is it more for documenting?
- What command(s) do I have to run to add a new story?
- What command(s) do I have to run to view the component "directory"?
- A link to a good storybook resource for the uninitiated would be nice :)

Reply:
Storybook is primarily documentation for designers. Stuff like commands to run it are better done in the README.md

This is essentially the home page when you load the storybook which describes what's available and you can see it if you run `npm run storybook` and yes it does have hot reloading.

My work colleague wrote a good summary of how useful storybook is here:

The stories are essentially examples of components taken in isolation.

https://medium.com/freely-sharing-the-sum-of-all-knowledge/storypops-or-the-benefits-of-component-isolation-with-storybook-936746a6c1a